packages/app/src/app/components/UploadProgress/Cube/elements.ts

```diff
@@ -9,35 +9,37 @@ const rotateAnimation = keyframes`
   }
 `;
 
-export const Cube = styled.div`
-  position: absolute;
-  width: ${props => props.size + 1}px;
-  height: ${props => props.size + 1}px;
-
-  * {
+export const Container = styled.div`
+  ${({ size }: { size: number }) => css`
     position: absolute;
-    width: ${props => props.size + 1}px;
-    height: ${props => props.size + 1}px;
-  }
+    width: ${size + 1}px;
+    height: ${size + 1}px;
+
+    * {
+      position: absolute;
+      width: ${size + 1}px;
+      height: ${size + 1}px;
+    }
+  `}
 `;
 
 export const Sides = styled.div`
-  ${({ noAnimation }) =>
-    !noAnimation &&
-    css`
-      animation: ${rotateAnimation} 1s linear infinite;
-    `};
-  transform-style: preserve-3d;
-  transform: rotateX(-37.5deg) rotateY(45deg);
+  ${({ size }: { size: number }) => css`
```

Comment:
@Saeris I think it's better to extract the `SidesProps` here, just to be consistent